package/MDAnalysis/lib/util.py

```diff
@@ -835,7 +825,7 @@ def guess_format(filename):
 
 def iterable(obj):
     """Returns ``True`` if *obj* can be iterated over and is *not* a  string."""
-    if isinstance(obj, basestring):
+    if isinstance(obj, (basestring, NamedStream)):
```

Comment:
That's workable but it means that `NamedStream` will only work as advertised within MDAnalysis . That's fine.

(I suppose for Py3 we will also need to change `basestring` here.)